examples/functions/timetable/Netex_51.1_Occupancy.xml

```diff
@@ -0,0 +1,1062 @@
+								<DayTypeRef version="any" ref="hde:DT_GoodFriday"/>
+								<DayTypeRef version="any" ref="hde:DT_EasterSunday"/>
+								<DayTypeRef version="any" ref="hde:DT_EasterMonday"/>
+							</dayTypes>
+							<ServicePatternRef version="any" ref="hde:svp_24o"/>
+							<TimeDemandTypeRef version="any" ref="mybus:tdt_01"/>
+							<BlockRef ref="mybus:BLK_24o5">EXTERNAL</BlockRef>
+							<LineRef version="any" ref="mybus:LN_24"/>
+							<JourneyPatternView>
+								<RouteRef version="any" ref="mybus:RT_24o"/>
+								<DirectionType>outbound</DirectionType>
+								<DestinationDisplayRef version="any" ref="mybus:DST_Charley"/>
+							</JourneyPatternView>
+							<runTimes>
+								<VehicleJourneyRunTime id="hde:vjrt_sj_24o_01">
+									<Name>Overall run time</Name>
+									<TimeDemandTypeRef version="any" ref="mybus:tdt_01"/>
+									<TimingLinkRef version="any" ref="mybus:SSP_001_to_SSP_077"/>
+									<RunTime>PT70M</RunTime>
+								</VehicleJourneyRunTime>
+							</runTimes>
+							<passingTimes>
+								<!--			01			02		 	 
+	A	d		14:00	 	15:00		    
+	  t1										+10m	RUN TIME 
+	  t2										+10m	RUN TIME 
+	B	a		14:30		15:30 		+10m	RUN TIME  
+	B	d		14:32	...	15:32 		+2m	 	WAIT TIME
+	  t1										+180m	 RUN TIME .
+	C	d		15:10		16:10		.+20m	RUN TIME  
+												+70m	 JOURNEY PATTERN RUN TIME  -->
+								<TimetabledPassingTime version="any" id="mybus:tpt_24o_01_SSP_001">
+									<StopPointInJourneyPatternRef version="any" ref="hde:spijp_24o_01"/>
+									<DepartureTime>14:00:00.0Z</DepartureTime>
+								</TimetabledPassingTime>
+								<TimetabledPassingTime version="any" id="hde:tpt_24o_01_SSP_001_t1">
+									<TimingPointInJourneyPatternRef version="any" ref="hde:tpijp_24o_02"/>
+									<DepartureTime>14:10:00.0Z</DepartureTime>
+									<occupancies>
+										<Occupancy>
+											<OccupancyLevel>fewSeatsAvailable</OccupancyLevel>
+										</Occupancy>
+									</occupancies>
+								</TimetabledPassingTime>
+								<TimetabledPassingTime version="any" id="hde:tpt_24o_01_SSP_001_t2">
+									<TimingPointInJourneyPatternRef version="any" ref="hde:tpijp_24o_03"/>
+									<DepartureTime>14:20:00.0Z</DepartureTime>
+								</TimetabledPassingTime>
+								<TimetabledPassingTime version="any" id="mybus:tpt_24o_01_SSP_002">
+									<StopPointInJourneyPatternRef version="any" ref="hde:spijp_24o_02"/>
+									<DepartureTime>14:30:00.0Z</DepartureTime>
+								</TimetabledPassingTime>
+								<TimetabledPassingTime version="any" id="hde:tpt_24o_01_SSP_002_t3">
+									<TimingPointInJourneyPatternRef version="any" ref="hde:tpijp_24o_05"/>
+									<DepartureTime>14:50:00.0Z</DepartureTime>
+									<occupancies>
+										<Occupancy>
+											<PassengerCategory>adult</PassengerCategory>
+											<OccupancyPercentage>60</OccupancyPercentage>
+										</Occupancy>
+										<OccupancyRef version="any" ref="occ4"/>
```

Comment:
What would this OccupancyRef mean in this context?

Reply:
Many Occupancy might be similar. So this is a way to reuse them.